docs/release-notes/NuGet-6.11.md

```diff
@@ -0,0 +1,101 @@
+---
+title: NuGet 6.11 Release Notes
+description: Release notes for NuGet 6.11 including new features, bug fixes, and DCRs.
+author: martinrrm
+ms.date: 8/1/2024
+ms.topic: conceptual
+---
+# NuGet 6.11 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.11**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.11](https://visualstudio.microsoft.com/downloads/) | [8.0.4xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.11
+
+* Suppress NuGetAudit warnings for specific advisories for PackageReference projects - [#13679](https://github.com/NuGet/Home/issues/13679)
+
+* Add `--allow-insecure-connections` option to dotnet SDK - [#13396](https://github.com/NuGet/Home/issues/13396)
+
+* Swap authors for owners in Visual Studio Package Manager UI - [#12501](https://github.com/NuGet/Home/issues/12501)
+
+* [Feature]: add dotnet nuget why to dotnet CLI - [#11943](https://github.com/NuGet/Home/issues/11943)
```

Comment:
Is this in .NET 8.0.4xx?

Reply:
yes, it is.

https://github.com/NuGet/NuGet.Client/commit/709d913595c39d14b98dbcadfdcc96bf34c712b0

https://github.com/NuGet/NuGet.Client/commits/6.11.0.86/